features/wishlists.feature

```diff
@@ -26,6 +26,37 @@ Scenario: Create a wishlist
     And the "Wishlist_Name" field should be empty
     And the "Wishlist_User_Id" field should be empty
     When I paste the "Wishlist_Id" field
-    And I press the "Search-Wishlist" button
+    And I press the "retrieve-Wishlist" button
```

Comment:
is it case sensitive? could you do `retrieve-wishlist` so it's the exact match to the button id please?

Reply:
I have already added status part below this line.